docs/explanation/gno-tooling/cli/gnokey.md

```diff
@@ -208,13 +208,17 @@ gnokey maketx call \
 
 #### **makeCallTx Options**
 
-| Name      | Type       | Description                      |
-| --------- | ---------- | -------------------------------- |
-| `send`    | String     | The amount of coins to send.     |
-| `pkgpath` | String     | The package path (required).     |
-| `func`    | String     | The contract to call (required). |
-| `args`    | String \[] | The arguments of the contract.   |
-
+| Name      | Type   | Description                               |
+| --------- | ------ |-------------------------------------------|
+| `send`    | String | The amount of coins to send.              |
+| `pkgpath` | String | The package path (required).              |
+| `func`    | String | The contract to call (required).          |
+| `args`    | String | An argument of the function being called. |
```

Comment:
Previously `args` had a type of string slice. But I could see how that could have been confusing since it is more variadic than a slice as it is coming from multiple command line args. So maybe there should be a way to indicate that, but I'm not sure how.

Reply:
My ideas with this was you can look at the example above and quite easily deduce that you can call `-args` multiple times. How about changing the description to:

 "An argument of the function being called. You can use `-args` multiple times in a single `call` command to accommodate possible multiple function arguments."

Reply:
[c062f9c](https://github.com/gnolang/gno/pull/1413/commits/c062f9c116f0b044202dce4cbf2956d4b9fd3aa5)